np.Preview/src/previewMain.js

```diff
@@ -82,41 +74,11 @@ Button a { text-decoration: none; font-size: 0.9rem; }
  * @author @jgclark
  * @param {string?} mermaidTheme name (optional)
  */
-export function previewNote(mermaidTheme?: string): void {
+export async function previewNote(mermaidTheme?: string): void {
```

Comment:
You've removed a default for mermaidTheme. Please add`='green'`.

Reply:
done